examples/kubernetes/servicemesh/cnp-with-mutual-auth.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: simple-policy
```

Comment:
super nit: give this a more descriptive name. "mutual-auth-echo" or something :-)